python/lsst/dbb/buffmngrs/handoff/cli.py

```diff
@@ -57,7 +57,7 @@ def initdb(filename):
     try:
         Base.metadata.create_all(engine)
     except (DBAPIError, SQLAlchemyError) as ex:
-        msg = f"cannot create tables: {ex}"
+        msg = "cannot create tables: %s" % (ex)
```

Comment:
This is still not doing lazy formatting. Changing to %-formatting still results in the string being formatted. It needs to be written as:

```
logger.error("Cannot create tables: %s", ex)
```
although consider instead doing:
```
logger.exception("Cannot create tables", exc_info=1)
```

Reply:
Also, since you are using the `msg` later explicitly in the `raise RuntimeError` below pre-filling the message with f-string style is fine.